.github/workflows/full-unittests.yml

```diff
@@ -0,0 +1,91 @@
+name: cellxgene_census package full unit tests
+
+# Run all unit tets, including those that are too expensive to run frequently.
+# This workflow requires a very large capacity runner, e.g., 1+TiB RAM. There
```

Comment:
```suggestion
# This workflow requires a very large capacity, self-hosted runner, e.g., 1+TiB RAM. There
```
(or mention in some comment that this depends on a self-hosted runner)

Reply:
we don't actually care if it is self-hosted or not, as it is a generic ubuntu image.  Just care if it has large resources